src/static/riot/datasets/management.tag

```diff
@@ -1,9 +1,82 @@
+                <option value="competition_bundle">Competition Bundle</option>
+            </select>
+            <div style="margin-left: 1vw;" class="ui checkbox input" onclick="{ filter }">
+                <label>Show Auto Created?</label>
+                <input type="checkbox" ref="auto_created">
+            </div>
+            <div style="margin-left: 30vw;" class="ui input">
+                <a class="ui green button" onclick="{show_dataset_modal}">Add Dataset</a>
+            </div>
+
+            <table class="ui celled compact table">
+                <thead>
+                <tr>
+                    <th>Name</th>
+                    <th width="175px">Type</th>
+                    <th width="125px">Uploaded...</th>
+                    <th width="125px">Key</th>
+                    <th width="50px">Public</th>
+                    <th width="50px">Delete?</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr each="{ dataset, index in filtered_datasets }" class="dataset-row {warning: dataset.was_created_by_competition}">
+                    <td><a href="{ URLS.DATASET_DOWNLOAD(dataset.key) }">{ dataset.name }</a></td>
+                    <td>{ dataset.type }</td>
+                    <td>{ timeSince(Date.parse(dataset.created_when)) } ago</td>
+                    <td>{ dataset.key } ago</td>
+                    <td class="center aligned">
+                        <i class="checkmark box icon green" show="{ dataset.is_public }"></i>
+                    </td>
+                    <td class="center aligned">
+                        <button class="mini ui button red icon" onclick="{ delete_dataset.bind(this, dataset) }">
+                            <i class="icon delete"></i>
+                        </button>
+                    </td>
+                </tr>
+                </tbody>
+                <tfoot>
+                <!-- Pagination that we may want later... -->
+                <tr>
+                    <th colspan="8">
+                        <div class="ui right floated pagination menu">
+                            <a show="{pagination.previous !== null}" class="icon item" onclick="{previous_page}">
+                                <i class="left chevron icon"></i>
+                            </a>
+                            <div class="item">
+                                <label>{dataset_page}</label>
+                            </div>
+                            <a show="{pagination.next !== null}" class="icon item" onclick="{next_page}">
+                                <i class="right chevron icon"></i>
+                            </a>
+                        </div>
+                    </th>
+                </tr>
+               <!-- -->
+                </tfoot>
+            </table>
+        </div>
+    </div>
+
 
-    <div class="ui two column grid">
+    <div id="dataset_modal" class="ui modal">
```

Comment:
Should we just use a `ref` instead of `id`? That way we don't have to do the jquery lookup each time, more riotjs-ee, etc.